src/Decompiler/Decompiler.cs

```diff
@@ -269,20 +270,24 @@ public void Assemble(string fileName, Assembler asm)
         /// <param name="fileName"></param>
         /// <param name="arch"></param>
         /// <param name="platform"></param>
-        public Program LoadRawImage(string fileName, LoadDetails raw)
+        public Program LoadRawImage(string fileName, string outputDir, LoadDetails raw)
         {
             eventListener.ShowStatus("Loading raw bytes.");
             byte[] image = loader.LoadImageBytes(fileName, 0);
             var program = loader.LoadRawImage(fileName, image, null, raw);
-            Project = CreateDefaultProject(fileName, program);
+            Project = CreateDefaultProject(fileName, outputDir, program);
             eventListener.ShowStatus("Raw bytes loaded.");
             return program;
         }
 
-        protected Project CreateDefaultProject(string fileName, Program program)
+        protected Project CreateDefaultProject(string fileNameWithPath, string outputDir, Program program)
```

Comment:
Maybe it's not best solution to add `output directory` parameter to `CreateDefaultProject`. We should create _default_ project in this method. So we need to auto-genetate some path (and maybe it should be done in `Program.EnsureFilenames`)